content/src/editor.less

```diff
@@ -6,6 +6,19 @@
   box-sizing: content-box
 }
 
+.tooltipster-shadow {
+  border-radius: 5px;
+  background-color: #eeeeee;
+  box-shadow: 0px 0px 14px rgba(0,0,0,0.3);
+  color: #2c2c2c;
+  .tooltipster-shadow .tooltipster-content {
```

Comment:
why .tooltipster-shadow inside .tooltipster-shadow?  Do we really nest the same class in itself in the HTML?